packages/react-scripts/scripts/build.js

```diff
@@ -141,3 +142,30 @@ function copyPublicFolder() {
     filter: file => file !== paths.appHtml,
   });
 }
+
+function formatError(err) {
+  const message = get(err, 'message');
+
+  // Add more helpful message for UglifyJs error
+  if (typeof message === 'string' && message.indexOf('from UglifyJs') !== -1) {
+    try {
+      console.log(
+        'UglifyJs could not parse the code from \n\n',
+        chalk.yellow(
+          err.stack.split('\n')[1].split('[')[1].split('][')[0].replace(']', '')
```

Comment:
my regex is not very good, the string from `err.stack.split('\n')[1]` looks like this 
```
Unexpected token: name (A) [./packages/react-scripts/~/asdf/index.js:1,0][static/js/main.6078c716.js:9605,6]
```
Haven't developed the muscle memory for regexing that 😢